deploy/chart/templates/0000_50_olm_18-upstream-operators.catalogsource.yaml

```diff
@@ -1,3 +1,5 @@
+#! validate-crd: ./deploy/chart/templates/06-catalogsource.crd.yaml
+#! parse-kind: CatalogSource
```

Comment:
I seem to remember these caused problems with the release artifacts, so we removed them.

Reply:
As far as I understand it, these lines will get stripped when a release is made. These lines were already there.